tests/debuginfo/function-names.rs

```diff
@@ -37,7 +37,7 @@
 // Const generic parameter
 // gdb-command:info functions -q function_names::const_generic_fn.*
 // gdb-check:[...]static fn function_names::const_generic_fn_bool<false>();
-// gdb-check:[...]static fn function_names::const_generic_fn_non_int<{CONST#ad91263f6d2dd96e}>();
+// gdb-check:[...]static fn function_names::const_generic_fn_non_int<{CONST#[...]}>();
```

Comment:
The debuginfo test suite has a much weaker facility than FileCheck's for pattern-matching, as it only supports this kind of elision. For now, that's fine.

Reply:
Apparently this is buggy.